Resources/Prototypes/Entities/Mobs/Species/slime.yml

```diff
@@ -27,6 +43,16 @@
   - type: Damageable
     damageContainer: Biological
     damageModifierSet: Slime
+  - type: Geras
+  - type: PassiveDamage # Around 8 damage a minute healed
+    allowedStates:
+    - Alive
+    damageCap: 35
+    damage:
+      types:
+        Heat: -0.14
+      groups:
+        Brute: -0.14
   - type: DamageVisuals
```

Comment:
Healing the 2 most common damage types in the game are kinda sus.

Reply:
To be fair, they are also weak to all of afformentioned damage types - but sure, I'll raise the damage cap